src/redux/fetch/booksFetcher.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-param-reassign */
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const base = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+const url = `${base}/apps/7DODvx5AcKK3GILYFT27/books`;
+// const url = `${base}/apps/9cmRMcVZ8DnPUoFM6AOr/books`;
```

Comment:
- [ Required ] Please consider removing any commented code from your code base, commented code makes your project unprofessional. It would be better if any code that is not necessary is removed from your code base.